yewtil/Cargo.toml

```diff
@@ -10,12 +10,19 @@ readme = "Readme.md"
 
 [features]
 default = ["stable"] # Only stable is included by default.
-all = ["stable", "experimental"]
+all = ["stable", "experimental", "soft_depricated"]
 
 # Broad features
-## All features MUST be stable or experimental
-stable = ["neq", "pure", "history", "mrc_irc", "effect", "future", "store"]
-experimental = ["dsl", "lrc", "with_callback", "fetch"]
+## All features MUST be stable or experimental or soft_depricated
+stable = ["neq", "history", "mrc_irc", "future", "store"]
+experimental = [ "fetch"]
+# The next release intends to remove these features
+# Please open an issue if removal of any of these would affect your project.
+soft_depricated = [
```

Comment:
Here too: `deprecated`

What is the difference between `soft_deprecated` and marking the relevant items as [`#[deprecated]`](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute)?

Putting them behind a feature-flag seems to be a rather hard way of doing this (as a straightforward update would not work)

Reply:
Good point. I'll opt to do that instead.